### PR TITLE
feature/pre-commit

### PR DESCRIPTION
**Pull request by sregales-TNO**
Annoyed with failing styling, because I forgot to check before push. This will prevent that headached

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    rev: v4.0.1
+    hooks:
+    -   id: no-commit-to-branch
+    -   id: check-merge-conflict
+    -   id: check-toml
+    -   id: check-yaml
+    -   id: end-of-file-fixer
+    -   id: trailing-whitespace
+
+-   repo: https://github.com/pycqa/isort
```
Comment (markusdregi) [inline .pre-commit-config.yaml:12]: I think we should discuss `isort` and its implications in a separate PR... Furthermore, I don't think we should add it as a pre-commit hook before it is part of our workflow checks.

Comment (sregales-TNO) [reply .pre-commit-config.yaml:12]: Understandable and agree

**Comment by markusdregi** [comment]
Is this something you would still like to pursue or get input on @sregales-TNO? Looks to be rather stale...

**Comment by oyvindeide** [comment]
Closing as inactive, feel free to reopen if that is not the case.

**Pull request closed without merging**